nearcore/src/config.rs

```diff
@@ -334,6 +334,33 @@ fn default_enable_rocksdb_statistics() -> bool {
     false
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct StoreConfig {
+    /// Maximum number of store files being opened simultaneously.
```

Comment:
let's also mention pros & cons of increasing/lowering.

(I guess for increasing we should also tell people to change the ulimit etc)